src/openrct2/core/Collections.hpp

```diff
@@ -65,46 +65,37 @@ namespace Collections
         return SIZE_MAX;
     }
 
-    #pragma region String helpers
+#pragma region String helpers
 
-    template<typename TCollection>
-    static bool Contains(TCollection &collection, const char * item, bool ignoreCase = false)
+    template<typename TCollection> static bool Contains(TCollection& collection, const char* item, bool ignoreCase = false)
     {
-        return Contains(collection, item,
-            [ignoreCase](const char * a, const char * b)
-            {
-                return String::Equals(a, b, ignoreCase);
-            });
+        return Contains(
+            collection, item, [ignoreCase](const char* a, const char* b) { return String::Equals(a, b, ignoreCase); });
```

Comment:
I thought we had `AllowShortFunctionsOnASingleLine` set to false, but apparently not. I hope it applies to lambdas too.

Reply:
Removing `ColumnLimit:     128` makes clang-format add a break after the `{`. Better, but still not 100% what we want. This may be a bug in clang-format.
I'll leave it for now, and check how removing the column limit affects the rest of the code later.

Reply:
After some more testing things out, it just looks like this happens because the default column width is 80 characters. This line goes just over it. clang-format then decides to break it after the opening brace. A similar thing happens with templated classes and arrays that are shorter than 80 characters in width.

I've gone though the options documentation again, but there's currently no way of always breaking lambda's, `template<...> class`, and short arrays. Hopefully these options get added later on.